contracts/schemes/GlobalConstraintRegistrar.sol

```diff
@@ -0,0 +1,187 @@
+pragma solidity ^0.5.11;
+
+import "@daostack/infra/contracts/votingMachines/IntVoteInterface.sol";
+import "@daostack/infra/contracts/votingMachines/VotingMachineCallbacksInterface.sol";
+import "../votingMachines/VotingMachineCallbacks.sol";
+import "@openzeppelin/upgrades/contracts/Initializable.sol";
+
+/**
+ * @title A scheme to manage global constraint for organizations
+ * @dev The scheme is used to register or remove new global constraints
+ */
+contract GlobalConstraintRegistrar is Initializable, VotingMachineCallbacks, ProposalExecuteInterface {
```

Comment:
I think we should make GC also non-universal, so it would be just constraints.

Reply:
it is none universal